simplemma/lemmatizer.py

```diff
@@ -106,27 +105,22 @@ def _decompose(
         # try other case
```

Comment:
The section below is not well-covered in terms of tests (my bad), so I'm reluctant to change it. I know the code is experimental and can be improved.

Reply:
If you look at lines 110 and 112, it's impossible that candidate is not None. So it makes sense to simply return.
Line 115 is a duplicate of line 100.
And line 121 is followed by a break & return so it makes sense to just return.

So, overall, I didn't change the logic.
Test coverage has increased 0.6% with this change and the only lines that seemed not covered by tests are 108-109 according to the codecov report.

It's up to you but I see this as a low-risk beneficial change 🙂 
And additional tests can always be added.

Reply:
I can't think of a good test so it's probably OK to change the lines.
I'll address the issue with the comments another time.